payments-core/api/payments-core.api

```diff
@@ -7754,6 +7754,16 @@ public final class com/stripe/android/view/ExpiryDateEditText : com/stripe/andro
 	public final fun isDateValid ()Z
 }
 
+public class com/stripe/android/view/LifecycleOwnerLayout : android/widget/LinearLayout, androidx/lifecycle/LifecycleOwner, androidx/lifecycle/ViewModelStoreOwner, androidx/savedstate/SavedStateRegistryOwner {
```

Comment:
Any way we can provide the `LifecycleOwnerLayout` functionality without having to make it a super class (and thus exposing it)?

Maybe it could be a delegate that we call in `onAttachedToWindow` and `onDetachedFromWindow`, so that our views would only need to implement `LifecycleOwner`, `ViewModelStoreOwner`, and `SavedStateRegistryOwner`.

Reply:
I think it's better to only expose things as public API if we actually want to support it as public API, because once we do this, I think we're kind of stuck with it. 

Till's suggestion sounds like it wouldn't complicate the card elements very much and I think it'd be worth it to not expose this. 

Also if we're exposing this as public API, wouldn't we need to do an API review?

Reply:
What if we add `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)` to LifecycleOwnerLayout, that way it won't be in the API dump.

Reply:
Updated to use`LifecycleOwnerDelegate` 👍